packages/config/test/unit/index_spec.js

```diff
@@ -146,4 +146,22 @@ describe('src/index', () => {
       })
     })
   })
+
+  describe('.validateNoReadOnlyConfig', () => {
+    it('returns an error if validation fails', () => {
+      const errorFn = sinon.spy()
+
+      configUtil.validateNoReadOnlyConfig({ chromeWebSecurity: false }, errorFn)
+
+      expect(errorFn).to.have.been.callCount(1)
```

Comment:
This might just work itself out, but I think you can lose the .been in the chain here (and below):
```js
expect(errorFn).to.have.callCount(1)
```

Reply:
Personally, I like `expect(errorFn).to.be.calledOnce`